netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -241,28 +243,35 @@ public NettyChannelBuilder usePlaintext(boolean skipNegotiation) {
     return this;
   }
 
-
   /**
-   * Enable keepalive with default delay and timeout.
+   * Sets a custom keepalive time, the delay time for sending next keepalive ping. An unreasonably
+   * small value might be increased, and {@code Long.MAX_VALUE} nano seconds or an unreasonably
+   * large value will disable keepalive.
+   *
+   * @since 1.3.0
    */
-  public final NettyChannelBuilder enableKeepAlive(boolean enable) {
```

Comment:
Please leave these for a cycle, to ease upgrading (even though we know that keepalive was broken). Just mark them as `@Deprecated`. They can be reimplemented by calling the new API if that makes it easier.